### ISSUE TITLE
Any further development plans for this project?

### ISSUE THREAD

**Issue by dzalf** [jonblack/arduino-fsm#43]
Dear Jon and contributors

I was in the midst of writing my own FSM library when I came across this repo.

However, I can see that it has not been updated in years. 

Are there any alternative forks that took over the maintenance of this project?

Otherwise, is anyone willing to keep it alive? I could personally do it however, I would need to find some space in my (very) busy schedule

Cheers

**Comment by mhaberler** [comment]
yeah it's a pity - pretty useful per se. I posted https://github.com/jonblack/arduino-fsm/pull/42 - no response

I concluded this project was abandoned so did my fork

there are quite a few useful improvements and fixes in the forks and PR's - I'd be interested in the context pointer feature

not prepared to take over maintenance but will re-post PR's and salvage the occasional feature/bugfix

- Michael

**Comment by dzalf** [comment]
Thanks, Michael

I totally understand that most of us have our hands busy, and possibly Jon isn't the exemption.

I will have a look at your fork. It might be a good idea to _Frankenstein_ some of the PRs (after thorough testing, of course) then upload it as a new fork.

Best wishes to you all

**Comment by jonblack** [comment]
Hi all. I'm really sorry that I haven't responded to any of the merge requests. I'm open to giving someone contributor permissions to work through some of them. Sadly, I'm unable to do this at the moment. Does that help?

**Comment by dzalf** [comment]
Hi Jon

No worries. Thank you for taking the time to respond to our concerns.

Let me organise my schedule and I will get back to you with a response to your offer. At the moment, I am in the middle of both professional and personal changes. Additionally, I would like to _fully_ understand the basic capabilities of the library first and perhaps test one or two of the proposed PRs so that I could _confidently_ approve them (or not)

In the meantime, perhaps another _brave_ contributor is willing to take the opportunity given by you?

Best wishes

Cheers
